docs/tutorials/podman_tutorial.md

```diff
@@ -7,7 +7,7 @@ commands with Podman.
 
 ## Install Podman on Fedora from RPM Repositories
 Fedora 27 and later provide Podman via the package manager.
-```
+```bash
```

Comment:
This gets you [Shell-script syntax highlighting][0], but what you have below (with its `$` prompt) is a [shell session][1].  I recommend `bash` -> [`console`][3] here.

[0]: https://github.com/github/linguist/blob/v6.2.0/lib/linguist/languages.yml#L4317-L4378
[1]: https://github.com/github/linguist/blob/v6.2.0/lib/linguist/languages.yml#L4379-L4390
[2]: https://github.com/github/linguist/blob/v6.2.0/lib/linguist/languages.yml#L4323
[3]: https://github.com/github/linguist/blob/v6.2.0/lib/linguist/languages.yml#L4385